contracts/src/v0.8/keystone/KeystoneForwarder.sol

```diff
@@ -49,9 +47,8 @@ contract KeystoneForwarder is IForwarder, ConfirmedOwner, TypeAndVersionInterfac
 
   /// @notice This error is thrown whenever a report specifies a configuration that
   /// does not exist.
-  /// @param donId The DON ID that was provided in the report
-  /// @param configVersion The config version that was provided in the report
-  error InvalidConfig(uint32 donId, uint32 configVersion);
+  /// @param configId (uint64(donId) << 32) | configVersion
```

Comment:
Does this packing make a non-negligible difference?

Reply:
It unpacks the two values as one when fetching from the metadata which is slightly less work. We should still extract donId/configVersion in the error case though, to make this error easier to read